test-e2e/utils/steps_file.ts

```diff
@@ -0,0 +1,116 @@
+import * as assert from 'assert';
+
+declare global {
+  let jwplayer: () => {getState: () => string};
+}
+
+module.exports = function() {
+  return actor({
+
+    // Define custom steps here, use 'this' to access default methods of I.
+    // It is recommended to place a general 'login' function here.
+
+    login: function (this: CodeceptJS.I, email = '12345@test.org', password = 'Ax854bZ!$') {
```

Comment:
Can we use the constants.username and password here?

Reply:
Done